package.json

```diff
@@ -222,4 +222,4 @@
     "business",
     "2fa"
   ]
-}
+}
```

Comment:
You removed the new line at the end of the file. Rollback.

Reply:
After one of upgrading attempts the wizard removed all packages from `package.json` and left only one, it is glitchy in some circumstances